lib/services/verification.service.js

```diff
@@ -6,10 +6,9 @@ import {Ed25519Signature2018} from "@digitalbazaar/ed25519-signature-2018";
 import {LDKeyPair as RSAKeyPair} from "crypto-ld";
 import {Ed25519Signature2020} from "@digitalbazaar/ed25519-signature-2020";
 import * as vc from "@digitalbazaar/vc";
-import { createRequire } from 'module';
```

Comment:
module is not available in browser. so, updated the import part from jsonld-signatures